source

```diff
@@ -100164,255 +100161,256 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
     <tr>
       <td>
       <dfn data-x="parse-error-absence-of-digits-in-numeric-character-reference">absence-of-digits-in-numeric-character-reference</dfn>
       <td><p>This error occurs if the parser encounters a numeric <span
       data-x="syntax-charref">character reference</span> that doesn't contain any digits (e.g. <code
-      data-x="">&amp;#qux;</code>). In that case parser doesn't resolve the character reference.</p>
+      data-x="">&amp;#qux;</code>). In this case the parser doesn't resolve the
+      <span data-x="syntax-charref">character reference</span>.</p>
     <tr>
       <td>
       <dfn data-x="parse-error-null-character-reference">null-character-reference</dfn>
       <td><p>This error occurs if the parser encounters a numeric <span
       data-x="syntax-charref">character reference</span> that references a U+0000 NULL character.
-      Parser resolves such character references to a U+FFFD REPLACEMENT CHARACTER character.</p>
+      The parser resolves such character references to a U+FFFD REPLACEMENT CHARACTER character.</p>
     <tr>
       <td>
       <dfn data-x="parse-error-non-unicode-character-reference">non-unicode-character-reference</dfn>
       <td><p>This error occurs if the parser encounters a numeric <span
       data-x="syntax-charref">character reference</span> that references a character that is not a
       Unicode character, i.e. isolated <span>surrogate</span> or code point that is greater than
-      valid Unicode range. Parser resolves such character references to a U+FFFD REPLACEMENT
+      valid Unicode range. The parser resolves such character references to a U+FFFD REPLACEMENT
       CHARACTER character.</p>
-    <tr>
+     <tr>
       <td>
       <dfn data-x="parse-error-undefined-character-reference">undefined-character-reference</dfn>
       <td><p>This error occurs if the parser encounters a numeric <span
       data-x="syntax-charref">character reference</span> that references a character that is a
-      permanently undefined Unicode character. Parser resolves such character references as is.</p>
+      permanently undefined Unicode character. The parser resolves such character references as is.</p>
     <tr>
       <td>
       <dfn data-x="parse-error-control-character-reference">control-character-reference</dfn>
       <td><p>This error occurs if the parser encounters a numeric <span
       data-x="syntax-charref">character reference</span> that references a Unicode <span
       data-x="control">control</span> character that is not an <span>ASCII whitespace</span>
-      character or is a U+000D CARRIAGE RETURN (CR) character. In general, parser resolves such
-      characters as is, except C1 controls that are replaced.</p>
+      character or U+000D CARRIAGE RETURN (CR) character. The parser resolves such
+      characters as is except C1 controls that are replaced.</p><!-- TODO: replaced with what? -->
```

Comment:
The are replaced according to the table in the [Numeric character reference end state](https://html.spec.whatwg.org/multipage/syntax.html#numeric-character-reference-end-state). Such replacements are used in various encodings and seems like it's a common approach for C1 controls handling. However, I haven't found any common term for such replacement technique.